Makefile

```diff
@@ -54,6 +54,7 @@ deploy-crds: $(DEPLOY_CRDS)
 deploy-objects: deploy-crds
 	for obj in $(DEPLOY_OBJECTS); do \
 		$(TEMPLATE_CMD) $$obj | kubectl apply -f - ;\
+		sleep 1;\
```

Comment:
I realize it's simple enough to work, but alternatively maybe do a wait?

Reply:
This is only used for manual deployment when developing and testing, the main install path is per  OLM. Do not want to put a lot of "logic" into this.